common/history/characters/0_dvg_iraq.txt

```diff
@@ -0,0 +1,21 @@
+﻿CHARACTERS = {

```

Comment:
[12:26:11][character_templates_utils.cpp:29]: Invalid Ideology ideology_zaydani in  file: common/history/characters/0_dvg_iraq.txt line: 3
[12:26:11][character_templates_utils.cpp:29]: Can only do leader Ideologies in  file: common/history/characters/0_dvg_iraq.txt line: 3
[12:26:11][jomini_effect.cpp:128]: PostValidate of effect 'create_character' returned false at  file: common/history/characters/0_dvg_iraq.txt line: 3
[12:26:11][jomini_script_system.cpp:263]: Script system error!
  Error: add_ideology effect [ Invalid database object 'paternalistic' ]
  Script location: file: common/history/countries/0_dvg_middle_east.txt line: 97

[12:26:11][jomini_effect.cpp:128]: PostValidate of effect 'add_ideology' returned false at  file: common/history/countries/0_dvg_middle_east.txt line: 97
[12:26:11][jomini_script_system.cpp:263]: Script system error!
  Error: has_template trigger [ Invalid database object 'agitator_louise_michel' ]
  Script location: file: common/scripted_effects; create_paris_commune_characters line: 131